### PR TITLE
nginx: expose all versions of deployments management API

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: common.nginx.conf [hunk 1]
```diff
@@ -135,7 +135,7 @@ location = /api/management/v1/deployments/artifacts {
 	proxy_set_header X-MEN-RequestID $requestid;
 }
 
-location ~ /api/management/v1/deployments(?<endpoint>/.*) {
+location ~ /api/management/(v[0-9]+)/deployments(?<endpoint>/.*) {
```
Comment (tranchitella) [inline common.nginx.conf:138]: Minor cosmetic issue, I don't think you need the parentheses

Comment (kjaskiewiczz) [reply common.nginx.conf]: @tranchitella fixed

**Comment by kjaskiewiczz** [comment]
@tranchitella the PR is now rebased this PR on top of master

**Comment by tranchitella** [comment]
@kjaskiewiczz thanks, merged!

**Pull request merged**